src/Loader.tsx

```diff
@@ -0,0 +1,3 @@
+export const Loader = () => (
+  <div>Loading...</div>
```

Comment:
The Loader component is missing a return type. Consider specifying the return type as React.FC for better typing.